src/containers/ContactManagement/ContactManagement.module.css

```
@@ -14,15 +15,18 @@
   flex-direction: column;
   justify-content: space-between;
   width: 50%;
-  max-height: 65%;
+  height: 600px;
   background-color: #fff;
   border-radius: 15px;
   padding: 1.5rem 2rem;
-  height: auto;
+  overflow-y: scroll;
+
+  scrollbar-width: none;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)